integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/ITUtils.java

```diff
@@ -53,5 +65,62 @@ public static void overwriteConfigFile(String configFileName, Properties props)
     }
   }
 
+  public static void assertionsTableInfo(
```

Comment:
Can Hive IT and Iceberg IT also reuse this utility method?

Reply:
The current content of the function `assertionsTableInfo` in `CatalogHiveIT` and `CatalogIcebergIT` differs from that in `ITUtils`; therefore, I don't think it can be reused.